sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/SequencePointCollectorTest.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * SonarSource :: .NET :: Shared library
+ * Copyright (C) 2014-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugins.dotnet.tests;
+
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import static org.mockito.Mockito.*;
+
+public class SequencePointCollectorTest {
+  @Test
+  public void givenEmptyListOfPoints_publishCoverage_doesNotAddBranchCoverage() {
+    Coverage coverage = mock(Coverage.class);
+
+    SequencePointCollector sut = new SequencePointCollector();
```

Comment:
sut = system under test (https://martinfowler.com/articles/mocksArentStubs.html)

Reply:
it's the first time I see it, you could extend the name and use `systemUnderTest`

Reply:
The acronym is largely used and I hope it will enter soon in our vocabulary. If you have a strong opinion, I can change it, but I prefer to keep it like this since it's simple and it keeps the code lines short.

Reply:
that's fine, no strong opinion here